macosx/InfoPeersViewController.mm

```diff
@@ -140,6 +140,13 @@ - (void)updateInfo
     NSUInteger ltep = 0;
     NSUInteger toUs = 0;
     NSUInteger fromUs = 0;
+    NSUInteger knownTracker = 0;
+    NSUInteger knownIncoming = 0;
+    NSUInteger knownCache = 0;
+    NSUInteger knownLpd = 0;
+    NSUInteger knownPex = 0;
+    NSUInteger knownDht = 0;
+    NSUInteger knownLtep = 0;
```

Comment:
I don't know Obj-C at all, so just take this as a question instead of a request for change.

Could all of this be an array instead?

Reply:
Well, it's ObjC++, meaning you could write almost all C++ code you want here. But to what benefit? We'd still need to declare each index with a meaningful name to avoid confusion:
```
int connectedIndex = 0;
int trackerIndex = 1;
int incomingIndex = 2;
...
```
(also doable as an enum)
So in the end, the same number of lines, and no performance changes.

Reply:
We do have such an enum exposed as public API. And you can use the last enum value as the array size too.
So I thought you could save some lines and code duplication here.

https://github.com/transmission/transmission/blob/1c18737e67f94f28807f5e1fd22fcb22effdfb7d/libtransmission/transmission.h#L1411-L1421

Reply:
I see you did use this enum in other places. As long as it works better for you not using it here, I'm completely fine with it.

Reply:
I see you did use this enum in other places. As long as it works better for you not using it here, I'm completely fine with it.

Reply:
Ah yes, I see. In my defense, I did it in February, so I had forgotten about this enum. :P